automatic/winpcap/tools/chocolateyInstall.ps1

```diff
@@ -1,8 +1,27 @@
 $ErrorActionPreference = 'Stop'
 
+$packageArgs = @{
+	packageName  = 'WinPcap'
+	fileFullPath = "$(Get-PackageCacheLocation)\WinPcapInstall.exe"
+	url          = 'https://www.winpcap.org/install/bin/WinPcap_4_1_3.exe'
```

Comment:
Can we look to embed the binary in the package? This binary is no longer updated, so it will stop it disappearing from the web and the package being unusable.

Reply:
I've changed it to embed the installer.

Reply:
Though, I didn't include the file itself in the PR/commits, since for other packages that embed it, they were not included in this repo either.